### PR TITLE
Update image location sponsors 2458

### LINKED ISSUE #2458: Update the image location for sponsors
### Overview
We need to refactor the location for the sponsors images so they become reusable for other components.

### Action Items
- [x] Delete the existing `assets/images/sponsors` directory along with its contents
- [x] Move the file directory `assets/images/about/sponsors` one level up so it becomes `assets/images/sponsors`
- [x] As a result of the above change, change the paths of the sponsor images in the following file:
    - [x] [sponsors.yml](https://github.com/hackforla/website/blob/gh-pages/_data/internal/sponsors.yml)
    - [x] [about-card-sponsors.html](https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-sponsors.html)
- [x] Ensure that the [sponsors section](https://www.hackforla.org/about/#sponsors) of the about page looks the same after the changes
- [x] Ensure that the images for the sponsors at the bottom of the [citizen engagement page](https://www.hackforla.org/citizen-engagement) look the same after the changes
- [x] Ensure that the sponsors (bottom of the page) in the [current homepage](https://www.hackforla.org/#) look the same after the changes

### Resources/Instructions
[sponsors.yml](https://github.com/hackforla/website/blob/gh-pages/_data/internal/sponsors.yml)
[Sponsors section of the About Page](https://www.hackforla.org/about/#sponsors)
[Citizen engagement page](https://www.hackforla.org/citizen-engagement)
[Sponsors images directory](https://github.com/hackforla/website/tree/gh-pages/assets/images/about/sponsors)
[Current homepage](https://www.hackforla.org/#)

### PR DESCRIPTION
**Pull request by JasonY188**
Fixes #2458

### What changes did you make and why did you make them ?

  -Delete the existing assets/images/sponsors directory along with its contents
  -Move the file directory assets/images/about/sponsors one level up so it becomes assets/images/sponsors
  -Changed the paths for:  sponsors.yml, about-card-sponsors.html, partners.yml, citizen-engagement.html
  -Updated the code in sponsors.html

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screen Shot 2022-04-26 Before
 website.

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screen Shot 2022-04-26 After

</details>

### CODE REVIEW

**Comment by luke-karis** [comment]
Edit: Sorry I wasn't able to finish the review by 3/20 :( I promise I will have this reviewed by Monday 3/21
Review ETA: End of Day 3/21
Availability: Monday 3/20 5pm - 7pm

**Comment by usaboo** [comment]
Ugh sorry for the delay ->
Review ETA: End of Day, Saturday (March 19th, 22)
Availability: Saturday(19th March, 22) -> 11AM - 1PM(EST)

**Comment by usaboo** [comment]
Hi Jason, after running your branch on Docker, I was able to verify that the sponsor section in the about page looks exactly the same as before.

However images at the bottom of the citizen-engagement page look slightly different. I suspect that may be because, the current Code For America logo image (/assets/images/sponsors/logo-cfa.svg) and you are trying to use (/assets/images/sponsors/code-for-america.svg) are fundamentally not same image with both having different dimension. This if you look carefully results in a slightly different layout with the Organizations We work with section in the citizen engagement page of your fork looking slightly expanded than it actually is. 

citizen-engagement-sponsors-current
citizen-engagement-sponsors-jason

Also at the bottom of the current home-page, the sponsor images do not look correct, they appear a little shrunk. Please refer to the images I have attached for reference --> the first image is the from the current home page and the second image is from the homepage of your fork.

current homepage-sponsors
Jason's fork's homepage-sponsors

The changes to the partners.yml and sponsors.yml files look correct.

I honestly don't know if you can solve this or not, I think it is worth bringing it up in the dev meeting on Tuesday(March 22nd) and hopefully we can come to a resolution.

**Review by usaboo — CHANGES_REQUESTED** [review]
Hi Jason, after running your branch on Docker, I was able to verify that the sponsor section in the about page looks exactly the same as before.

However images at the bottom of the citizen-engagement page look slightly different. I suspect that may be because, the current Code For America logo image (/assets/images/sponsors/logo-cfa.svg) and you are trying to use (/assets/images/sponsors/code-for-america.svg) are fundamentally not same image with both having different dimension. This if you look carefully results in a slightly different layout with the ***Organizations We work With*** section in the ***Citizen Engagement*** page of your fork looking slightly expanded than it actually is. Please refer to the images I have attached below --> the first one is the current website look and the second one is from your fork.

citizen-engagement-sponsors-current
citizen-engagement-sponsors-jason

Also at the bottom of the current home-page, the sponsor images do not look correct, they appear a little shrunk. Please refer to the images I have attached for reference --> the first image is the from the current home page and the second image is from the homepage of your fork.

current homepage-sponsors
Jason's fork's homepage-sponsors

The changes to the partners.yml and sponsors.yml files look correct.

I honestly don't know if you can solve this or not, I think it is worth bringing it up in the dev meeting on Tuesday(March 22nd) and hopefully we can come to a resolution.

**Review by luke-karis — CHANGES_REQUESTED** [review]
Hey @JasonY188 thanks for working on this issue! I have noticed some of the same issues that Utkarsh indicated with his review when running your code locally. I will give this another look tomorrow before the weekly meeting and add more comments if need be.

**Comment by luke-karis** [comment]
One minor comment I have unrelated to your code is about your commit messages. It's generally a good idea to make each commit message contain a little information about the change so that you can easily go back and see what was changed when.

**Comment by JasonY188** [comment]
Thanks @luke-karis  and @usaboo  for reviewing my pr.  After our discussion, I will try to resized the images. 
ETA 03-25
Availability 4hrs

**Comment by SAUMILDHANKAR** [comment]
@JasonY188 Just wanted to follow up if you were able to resize the images or if it was too much hassle. Feel free to request a re-review based on your findings. Also, request you to share an update on your [pre-work checklist](https://github.com/hackforla/website/issues/2750) as well. Thank you.

**Comment by JasonY188** [comment]
@SAUMILDHANKAR I tried resizing the svg, but I'm not getting the desired result.  When I tried increasing the size of the image, I am only seeing changes in the sponsors section of the about page, not in the home page.  I will keep trying and provide an update on 03-29.

**Comment by JessicaLucindaCheng** [comment]
Leaving a note per our discussion on Tue, Mar 29: try editing the SCSS files related to the images to try to fix sizes of the images.

**Comment by SAUMILDHANKAR** [comment]
@JasonY188 Just wanted to let you know that we discussed this PR in our Dev/PM [agenda](https://github.com/hackforla/website/issues/2607#issuecomment-1086711156). As per the discussion, it is fine to use any image format (svg, png, jpeg etc) as long as the pages look fine. So, feel free to make the changes. Please let us know if there are any updates. Thank you.

**Comment by JasonY188** [comment]
Hi @SAUMILDHANKAR,

Thanks for the update.  I made a change in the _home.scss file so the svg images are a  bit larger now.  I will try using the previous images again and make a comparison to see which is better look.

ETA: 04-11
Availability: 4hrs

Screen Shot 2022-04-07 at 10 09 18 PM
.

**Comment by kathrynsilvaconway** [comment]
ETA 04/10 by 10pm EST
Availability 2hrs
**Note: I will review this this weekend if @JasonY188 is able to make the changes before Sunday night. If not, I will review it as soon as they are made.

**Comment by JasonY188** [comment]
@SAUMILDHANKAR 

The issue I found is that the original code uses two different sets of sponsor images, one for the home page and one for the about page.  Since the images are not identical, if we use the home page sponsor images, the about page images will be off and vice versa.  Even adjusting the scss, I am not able to make the images look like before.  On April 7th, the screenshot is the changed made on the sponsor section of home page.  Today's screenshot is the change made on the sponsor's section of the about page.

Please let me know which is preferred, or if you have any other suggestions.

Screen Shot 2022-04-10 at 7 15 08 PM

**Comment by kathrynsilvaconway** [comment]
@JasonY188 The shapes of some of the logos are distorted in the screen shot you posted today, so given a choice of the two, the ones from April 7th look better:
logos_apr_7
logos_apr_11
Looking at a local version you code, I can't see the images on the home page because the links are broken, so it is difficult to compare:
image

**Comment by JessicaLucindaCheng** [comment]
@kathrynsilvaconway I sometimes get broken links for images like that. You can try refreshing the page to see if that helps.

**Comment by SAUMILDHANKAR** [comment]
@JasonY188 Hi Jason, we discussed your blocker in our All-team meeting on Sunday and Bonnie suggested that the homepage sponsor images should look like the images on the About page. We can try using/copying the code from the about page for the images and see if that works.
@usaboo Please feel free to add your comments as well, if required.

**Comment by JasonY188** [comment]
Hi @SAUMILDHANKAR

Thanks for the update.  I will try your suggestion and let you know.

ETA 04-24
Availability 4hrs

**Comment by luke-karis** [comment]
Re-Review:
ETA: 5/2/22
Availability: Thursday 1hr, Saturday and Sunday 1-2 hrs

**Review by luke-karis — CHANGES_REQUESTED** [review]
Hey Jason, great job sticking with this, you're making great progress and the images look way better! I have two minor comments:

I noticed that there is some more spacing on the citizen engagement section you worked on.
Here is the original:
old_Citizen_engagement png
And here is what shows up when I test your changes locally 
new_Citizen_engagement

My other comment is I noticed that in the about page in the HfLA in the news section the CNN and GT images and links are swapped. Could you merge in the most recent website changes and make sure this looks good? It doesn't seem like you touched this code so I hope it's something funky with my computer and not a weird bug.

Current ordering:
Screenshot from 2022-05-02 22-01-59

Ordering when I run your branch locally:
Screenshot from 2022-05-02 22-02-57

**Comment by JessicaLucindaCheng** [comment]
@luke-karis @JasonY188 I looked at the code for the Press and I think the order of the Press has to do with the way Jekyll process the for loop in the Liquid templating language. Thus, for now, it is fine if the order is different from how you see it locally vs on the live website for the Press section.

**Comment by answebdev** [comment]
Availability: 2 hours
Review ETA: 5/6/2022 (end of day)

**Review by answebdev — COMMENTED** [review]
@JasonY188 Just wanted to check in on the current status, particularly on the "Organizations We work With" section in the "Citizen Engagement" page. Like @luke-karis , I also see a difference in spacing locally - there is more spacing in yours (see second image below):

<details>
  <summary>Original</summary>

organizations-live

</details>

<details>
  <summary>Locally</summary>

organizations-local

</details>

Secondly, like @luke-karis , I also noticed that in the About Page in the "HfLA in the News" section, the CNN and GT images and links are swapped. Though, according to the last comment made by @JessicaLucindaCheng , this is okay for now.

<details>
  <summary>Original</summary>

HfLA in the News_original

</details>

<details>
  <summary>Locally</summary>

HfLA in the News_locally

</details>

As for the Sponsors section on both the Home Page and About Page, the issues mentioned above seem to have been addressed, as they look fine for me locally now (if the images look off a little, it's because of the way I cropped them - they actually look fine online).

<details>
  <summary>Sponsors Home Page - Original</summary>

sponsors_home_original

</details>

<details>
  <summary>Sponsors Home Page - Locally</summary>

sponsors_home_locally

</details>

<details>
  <summary>Sponsors About Page - Original</summary>

sponsors_about_original

</details>

<details>
  <summary>Sponsors About Page - Locally</summary>

sponsors_about_locally

</details>

So basically, I'm wondering what the current status of the "Organizations We work With" section in the **Citizen Engagement** page is.

**Comment by usaboo** [comment]
Hi @JasonY188 , thanks for your work on this issue. I am so sorry I haven't been as prompt about my reviews. I have been traveling over the past 2 weeks.

However I do want to add something, @answebdev, @luke-karis , the sponsor section on homepage does not look for me in the deployed branch and Jason's branch, which is not what you see Adolf.

This is what I see on the deployed website's home-page,
on-the-web

This is what I see in Jason's branch's home-page running on my docker,
jason's-branch

Honestly, I feel it looks better in Jason's branch but that's for another day. I was wondering if I was doing everything right, or if we missed something.

**Comment by JasonY188** [comment]
Thank you all for reviewing my pr.  In regards to the Citizen-Engagement page in the Organization We Work With section, it's the same issue I was facing with the Sponsors section.  The images being replaced are not the same in size or resolution as images replacing them.   So with the new images there will be visual differences.  I'll look into adding css to the single image for Code for America, but I'm not sure that is the way to go.

Original Image
Screen Shot 2022-05-06 at 9 15 50 PM

Local Image
Screen Shot 2022-05-06 at 9 17 37 PM
 
ETA 05-10
Availability 4hrs

**Comment by JessicaLucindaCheng** [comment]
@JasonY188 You could ask Design at Sunday's meeting before you make additional SCSS changes to see what they think and give you feedback.

**Comment by answebdev** [comment]
@usaboo I don't know why you are not able to see the same thing as me. Sometimes I have issues where I am not able to see changes locally. In this case, it has been suggested to clear the browser's cache, though I have not actually tried doing this before. My understanding is that sometimes the file watching gets messed up and you need to rebuild everything. Saumil has suggested before deleting the __site folder_, deleting the _.jekyll-metadata_ file, and then building it back up from scratch. I believe I've tried this before, and it worked for me. If using Chrome, I found that it has also worked in incognito mode. These are some things you may want to try, if it is in fact the case of not being able to see your changes locally.

**Comment by JasonY188** [comment]
@JessicaLucindaCheng.  I could not attend last Sunday's meeting and most likely cannot attend this coming Sunday's meeting.  Is there an alternative?

**Comment by JasonY188** [comment]
@JessicaLucindaCheng @SAUMILDHANKAR  After Sunday's discussion, in the Citizen-Engagement page in the Organizations We Work With section, I will make SCSS changes to the Code for America image and reduce the gap between the images of the top and bottom rows to mirror the appearance of the live page.

ETA 05-21
Availability 4hrs

**Comment by JessicaLucindaCheng** [comment]
@JasonY188
> @JessicaLucindaCheng. I could not attend last Sunday's meeting and most likely cannot attend this coming Sunday's meeting. Is there an alternative?

I'm so sorry I missed your question. If you have a question in the future and I don't respond, please Slack message me.

**Comment by usaboo** [comment]
ETA - Thursday 11PM PT
Availability - 1 hour

**Comment by answebdev** [comment]
ETA - 5/25/2022 (end of day)
Availability - 2 hours

**Review by answebdev — COMMENTED** [review]
@JasonY188 Thank you for your updates. I checked the code and the updates you made, and I was able to see the changes you made in the code locally. As for the "Organizations We work With" section in the "Citizen Engagement" page, the extra spacing that was there previously is no longer there, and it looks better now.

<details>
  <summary>Locally</summary>

citizen-engagement-jason

</details>

<details>
  <summary>Website</summary>

citizen-engagement_website

</details>

As for the About Page in the "HfLA in the News" section, the CNN and GT images and links are still swapped. However, as was previously commented, this is okay for now.

As for the Sponsors section on the Home Page, although it now looks like the proposed changes and matches the screenshot of the visuals after changes are applied (see initial comment in PR), it does not look the same as it currently does live, and in the original issue, the last action item states to "Ensure that the sponsors (bottom of the page) in the current homepage look the same after the changes", so I'm not exactly sure which of the two is the way it is supposed to be (unless I misunderstood something). What do you think, @JessicaLucindaCheng ?

<details>
  <summary>Sponsors Home Page - Locally</summary>

home-sponsors-jason

</details>

<details>
  <summary>Sponsors Home Page - Website</summary>

home-sponsors-website

</details>

Finally, on the About Page, the images look the same as they previously did, and the same as on the live site.

<details>
  <summary>Sponsors About Page - Locally</summary>

about-sponsors-jason

</details>

<details>
  <summary>Sponsors About Page - Website</summary>

about-sponsors-website

</details>

**Comment by JessicaLucindaCheng** [comment]
@answebdev

> As for the Sponsors section on the Home Page, although it now looks like the proposed changes and matches the screenshot of the visuals after changes are applied (see initial comment in PR), it does not look the same as it currently does live, and in the original issue, the last action item states to "Ensure that the sponsors (bottom of the page) in the current homepage look the same after the changes", so I'm not exactly sure which of the two is the way it is supposed to be (unless I misunderstood something). What do you think, @JessicaLucindaCheng ?

As mentioned in https://github.com/hackforla/website/pull/2984#issuecomment-1101908974, the homepage sponsors' images should now look like the ones on the about page. Thus, the changes that Jason made to the sponsor images on the homepage are fine.

**Comment by usaboo** [comment]
Good job on your consistent work on this issue @JasonY188. As far as i can see,

-  the Sponsor section images in the homepage look fine ~ if they are supposed to be like the ones on about page.
- Next the spacing between the organizations we work with section images in citizen engagement section looks fixed now.

So everything looks good me.

Maybe others could find some bug that I may have missed.

**Review by usaboo — APPROVED** [review]

**Review by answebdev — APPROVED** [review]
@JessicaLucindaCheng Thank you for clearing that up. I think I either missed that comment, or just understand something else, so thank you for the clarification. @JasonY188  With that cleared up, everything else looks good to me now. Great job!

**Review by luke-karis — APPROVED** [review]
This looks good to me! Thanks for your continued work on this, you've done a great job pushing through and getting this done.
